Dockerfile.v2

```diff
@@ -0,0 +1,63 @@
+FROM radixdlt/babylon-node-build-layers:v1.3.0-java AS java-container
+
+WORKDIR /radixdlt
+
+ARG WGET_VERSION="1.21.3-1+b2"
+ARG VERSION_BRANCH=""
+ARG VERSION_COMMIT=""
+ARG VERSION_DISPLAY=""
+ARG VERSION_BUILD=""
+ARG VERSION_TAG=""
+ARG VERSION_LAST_TAG=""
+
+ENV VERSION_BRANCH=$VERSION_BRANCH
+ENV VERSION_COMMIT=$VERSION_COMMIT
+ENV VERSION_DISPLAY=$VERSION_DISPLAY
+ENV VERSION_BUILD=$VERSION_BUILD
+ENV VERSION_TAG=$VERSION_TAG
+ENV VERSION_LAST_TAG=$VERSION_LAST_TAG
+
+# Copy the relevant files at the repo root
+COPY \
+  build.gradle \
+  gradlew \
+  gradlew.bat \
+  settings.gradle \
+  sonar-project.properties \
+  gradle.properties \
+  licence-header.txt \
+  /radixdlt/
+COPY ./gradle /radixdlt/gradle
+COPY ./common /radixdlt/common
+COPY ./core /radixdlt/core
+COPY ./core-rust-bridge /radixdlt/core-rust-bridge
+COPY ./cli-tools /radixdlt/cli-tools
+COPY ./shell /radixdlt/shell
+COPY ./keygen /radixdlt/keygen
+
+RUN SKIP_NATIVE_RUST_BUILD=TRUE ./gradlew clean build -x test -Pci=true -PrustBinaryBuildType=release
+
+RUN cd core/build/distributions && \
+    unzip -j *.zip && \
+    mkdir -p /artifacts && \
+    cp *.jar /artifacts && \
+    cp core /artifacts
+
+FROM radixdlt/babylon-node-build-layers:v1.3.0-rust AS rust-container
```

Comment:
Minor - is it possible to move these versions together to variables at the top of this file? That way it's easier to update them all and not accidentally miss one :D